modules/apps/layout/layout-test/src/testIntegration/java/com/liferay/layout/search/test/LayoutMultiLanguageSearchTest.java

```diff
@@ -84,6 +85,24 @@ public void testJapaneseTitle() throws Exception {
 		_testLocaleKeywords(LocaleUtil.JAPAN, _JAPANESE_KEYWORD, _TITLE);
 	}
 
+	@Test
+	public void testLocalizedFields() throws Exception {
+		Locale locale = LocaleUtil.US;
+
+		setTestLocale(locale);
+
+		_addLayoutMultiLanguage();
+
+		Document document = layoutIndexerFixture.searchOnlyOne(
+			_ENGLISH_KEYWORD, locale);
```

Comment:
Trying to preview what Bchan would change maybe it is a good idea to Inline `locale` in both usages?

Reply:
Did it, what do you think now @gustavolimav?